{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Fetching gene sequence"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez\n",
    "from Bio import SeqIO\n",
    "Entrez.email = \"A.N.Other@example.com\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "gene_id = '1007358254'\n",
    "handle = Entrez.efetch(id=gene_id, db='nucleotide', rettype='gb', retmode='text')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "res = SeqIO.read(handle, format='gb')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_per_letter_annotations': {},\n",
       " '_seq': Seq('GGGATCAGAAAACTCGCTGGTTAATGAGTGGCCAAGAACATATCAGCTTAAGTT...CAA', IUPACAmbiguousDNA()),\n",
       " 'annotations': {'accessions': ['NM_001321072', 'XM_011529784'],\n",
       "  'comment': \"REVIEWED REFSEQ: This record has been curated by NCBI staff. The\\nreference sequence was derived from AP001630.1.\\nOn Mar 18, 2016 this sequence version replaced XM_011529784.1.\\nSummary: The protein encoded by this gene acts as a homotetramer to\\ncatalyze the conversion of homocysteine to cystathionine, the first\\nstep in the transsulfuration pathway. The encoded protein is\\nallosterically activated by adenosyl-methionine and uses pyridoxal\\nphosphate as a cofactor. Defects in this gene can cause\\ncystathionine beta-synthase deficiency (CBSD), which can lead to\\nhomocystinuria. This gene is a major contributor to cellular\\nhydrogen sulfide production. Multiple alternatively spliced\\ntranscript variants have been found for this gene. [provided by\\nRefSeq, Feb 2016].\\nTranscript Variant: This variant (5) initiates translation at an\\nalternate start codon compared to variant 1.\\nSequence Note: The RefSeq transcript and protein were derived from\\ngenomic sequence to make the sequence consistent with the reference\\ngenome assembly. The genomic coordinates used for the transcript\\nrecord were based on alignments.\\nPublication Note:  This RefSeq record includes a subset of the\\npublications that are available for this gene. Please see the Gene\\nrecord to access additional publications.\\n                               SAMEA1965299, SAMEA1966682\\n                               [ECO:0000350]\\nCOMPLETENESS: complete on the 3' end.\",\n",
       "  'data_file_division': 'PRI',\n",
       "  'date': '16-APR-2018',\n",
       "  'keywords': ['RefSeq'],\n",
       "  'molecule_type': 'mRNA',\n",
       "  'organism': 'Homo sapiens',\n",
       "  'references': [Reference(title='Upregulation of Cystathionine-beta-Synthase in Colonic Epithelia Reprograms Metabolism and Promotes Carcinogenesis', ...),\n",
       "   Reference(title='The c.797 G>A (p.R266K) cystathionine beta-synthase mutation causes homocystinuria by affecting protein stability', ...),\n",
       "   Reference(title='Augmented H2S production via cystathionine-beta-synthase upregulation plays a role in pregnancy-associated uterine vasodilation', ...),\n",
       "   Reference(title='Polymorphisms in MTHFR, MS and CBS genes and premature acute myocardial infarction in a Pakistani population', ...),\n",
       "   Reference(title='Homocystinuria Caused by Cystathionine Beta-Synthase Deficiency', ...),\n",
       "   Reference(title='Hereditary Dystonia Overview', ...),\n",
       "   Reference(title='Screening for mutations by expressing patient cDNA segments in E. coli: homocystinuria due to cystathionine beta-synthase deficiency', ...),\n",
       "   Reference(title='Purification and properties of cystathionine beta-synthase from human liver. Evidence for identical subunits', ...),\n",
       "   Reference(title='The molecular defect in a case of (cystathionine beta-synthase)-deficient homocystinuria', ...),\n",
       "   Reference(title='Cystathionine beta-synthase deficiency: a qualitative abnormality of the deficient enzyme modified by vitamin B6 therapy', ...)],\n",
       "  'sequence_version': 1,\n",
       "  'source': 'Homo sapiens (human)',\n",
       "  'structured_comment': OrderedDict([('Evidence-Data',\n",
       "                OrderedDict([('Transcript exon combination',\n",
       "                              'L00972.1 [ECO:0000332]'),\n",
       "                             ('RNAseq introns',\n",
       "                              'mixed/partial sample support')]))]),\n",
       "  'taxonomy': ['Eukaryota',\n",
       "   'Metazoa',\n",
       "   'Chordata',\n",
       "   'Craniata',\n",
       "   'Vertebrata',\n",
       "   'Euteleostomi',\n",
       "   'Mammalia',\n",
       "   'Eutheria',\n",
       "   'Euarchontoglires',\n",
       "   'Primates',\n",
       "   'Haplorrhini',\n",
       "   'Catarrhini',\n",
       "   'Hominidae',\n",
       "   'Homo'],\n",
       "  'topology': 'linear'},\n",
       " 'dbxrefs': [],\n",
       " 'description': 'Homo sapiens cystathionine-beta-synthase (CBS), transcript variant 5, mRNA',\n",
       " 'features': [SeqFeature(FeatureLocation(ExactPosition(0), ExactPosition(2656), strand=1), type='source'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(0), ExactPosition(2656), strand=1), type='gene'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(0), ExactPosition(627), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(614), ExactPosition(617), strand=1), type='misc_feature'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(626), ExactPosition(1967), strand=1), type='CDS'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(627), ExactPosition(762), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(762), ExactPosition(842), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(842), ExactPosition(977), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(977), ExactPosition(1047), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1047), ExactPosition(1139), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1139), ExactPosition(1265), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1265), ExactPosition(1350), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1350), ExactPosition(1456), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1456), ExactPosition(1534), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1534), ExactPosition(1669), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1669), ExactPosition(1778), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1778), ExactPosition(1863), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(1863), ExactPosition(2656), strand=1), type='exon'),\n",
       "  SeqFeature(FeatureLocation(ExactPosition(2655), ExactPosition(2656), strand=1), type='polyA_site')],\n",
       " 'id': 'NM_001321072.1',\n",
       " 'name': 'NM_001321072'}"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "acc_ids = res.annotations['accessions'] \n",
    "res.__dict__\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [],
   "source": [
    "features = {}\n",
    "for f in res.features:\n",
    "    features[f.type] = features.setdefault(f.type, []) + [f.__dict__]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [],
   "source": [
    "row = 0\n",
    "row_type = {}\n",
    "for f_type in features:\n",
    "    features[f_type].sort(key=lambda f: f['location']._end)\n",
    "    prev_end = -1\n",
    "    contiguous = 0\n",
    "    row_span = 1\n",
    "    for f in features[f_type]:\n",
    "        start, end = f['location']._start, f['location']._end\n",
    "        if start == prev_end:\n",
    "            contiguous += 1\n",
    "            row_span = 2\n",
    "        else:\n",
    "            contiguous = 0\n",
    "        f['coordinates'] = [(int(start), row + (contiguous % 2)), (int(end), row + (contiguous % 2))]\n",
    "        prev_end = end\n",
    "    row_type[row] = f_type\n",
    "    if row_span == 2:\n",
    "        row_type[row+1] = f_type\n",
    "    row += row_span\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'CDS': [{'coordinates': [(626, 3), (1967, 3)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(626), ExactPosition(1967), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('EC_number', ['4.2.1.22']),\n",
       "                ('note',\n",
       "                 ['isoform 2 is encoded by transcript variant 5; serine sulfhydrase; beta-thionase; methylcysteine synthase']),\n",
       "                ('codon_start', ['1']),\n",
       "                ('product', ['cystathionine beta-synthase isoform 2']),\n",
       "                ('protein_id', ['NP_001308001.1']),\n",
       "                ('db_xref', ['GeneID:875', 'HGNC:HGNC:1550', 'MIM:613381']),\n",
       "                ('translation',\n",
       "                 ['MAKCEFFNAGGSVKDRISLRMIEDAERDGTLKPGDTIIEPTSGNTGIGLALAAAVRGYRCIIVMPEKMSSEKVDVLRALGAEIVRTPTNARFDSPESHVGVAWRLKNEIPNSHILDQYRNASNPLAHYDTTADEILQQCDGKLDMLVASVGTGGTITGIARKLKEKCPGCRIIGVDPEGSILAEPEELNQTEQTTYEVEGIGYDFIPTVLDRTVVDKWFKSNDEEAFTFARMLIAQEGLLCGGSAGSTVAVAVKAAQELQEGQRCVVILPDSVRNYMTKFLSDRWMLQKGFLKEEDLTEKKPWWWHLRVQELGLSAPLTVLPTITCGHTIEILREKGFDQAPVVDEAGVILGMVTLGNMLSSLLAGKVQPSDQVGKVIYKQFKQIRLTDTLGRLSHILEMDHFALVVHEQIQYHSTGKSSQRQMVFGVVTAIDLLNFVAAQERDQK'])]),\n",
       "   'type': 'CDS'}],\n",
       " 'exon': [{'coordinates': [(0, 1), (627, 1)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(0), ExactPosition(627), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(627, 2), (762, 2)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(627), ExactPosition(762), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(762, 1), (842, 1)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(762), ExactPosition(842), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(842, 2), (977, 2)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(842), ExactPosition(977), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(977, 1), (1047, 1)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(977), ExactPosition(1047), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1047, 2), (1139, 2)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1047), ExactPosition(1139), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1139, 1), (1265, 1)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1139), ExactPosition(1265), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1265, 2), (1350, 2)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1265), ExactPosition(1350), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1350, 1), (1456, 1)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1350), ExactPosition(1456), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1456, 2), (1534, 2)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1456), ExactPosition(1534), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1534, 1), (1669, 1)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1534), ExactPosition(1669), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1669, 2), (1778, 2)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1669), ExactPosition(1778), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1778, 1), (1863, 1)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1778), ExactPosition(1863), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'},\n",
       "  {'coordinates': [(1863, 2), (2656, 2)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(1863), ExactPosition(2656), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('inference', ['alignment:Splign:2.1.0'])]),\n",
       "   'type': 'exon'}],\n",
       " 'gene': [{'coordinates': [(0, 4), (2656, 4)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(0), ExactPosition(2656), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('note', ['cystathionine-beta-synthase']),\n",
       "                ('db_xref', ['GeneID:875', 'HGNC:HGNC:1550', 'MIM:613381'])]),\n",
       "   'type': 'gene'}],\n",
       " 'misc_feature': [{'coordinates': [(614, 6), (617, 6)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(614), ExactPosition(617), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']),\n",
       "                ('gene_synonym', ['HIP4']),\n",
       "                ('note', ['upstream in-frame stop codon'])]),\n",
       "   'type': 'misc_feature'}],\n",
       " 'polyA_site': [{'coordinates': [(2655, 0), (2656, 0)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(2655), ExactPosition(2656), strand=1),\n",
       "   'qualifiers': OrderedDict([('gene', ['CBS']), ('gene_synonym', ['HIP4'])]),\n",
       "   'type': 'polyA_site'}],\n",
       " 'source': [{'coordinates': [(0, 5), (2656, 5)],\n",
       "   'id': '<unknown id>',\n",
       "   'location': FeatureLocation(ExactPosition(0), ExactPosition(2656), strand=1),\n",
       "   'qualifiers': OrderedDict([('organism', ['Homo sapiens']),\n",
       "                ('mol_type', ['mRNA']),\n",
       "                ('db_xref', ['taxon:9606']),\n",
       "                ('chromosome', ['21']),\n",
       "                ('map', ['21q22.3'])]),\n",
       "   'type': 'source'}]}"
      ]
     },
     "execution_count": 72,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "features"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Fetch variations from ClinVar"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [],
   "source": [
    "handle = Entrez.esearch(term=' OR '.join(['%s[Nucleotide/Protein Accession]' % acc_id for acc_id in acc_ids]), db='clinvar')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['519588', '370129', '340091', '264609', '236937', '212883', '212874', '189185', '188989', '188988', '136677']"
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "res = Entrez.read(handle, validate=False)\n",
    "var_ids = res['IdList']\n",
    "var_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "metadata": {},
   "outputs": [],
   "source": [
    "handle = Entrez.efetch(id=var_ids, db='clinvar', rettype='variation')\n",
    "# https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=clinvar&id=11%2C9&rettype=variation&tool=biopython"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\\n                        \\n<ClinVarResult-Set><VariationReport VariationID=\"519588\" VariationName=\"NM_000071.2(CBS):c.297C&gt;T (p.Phe99=)\" DateCreated=\"2018-04-12\" VariationType=\"Simple\" DateLastUpdated=\"2018-04-14\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"510862\">\\n    <Name>NM_000071.2(CBS):c.297C&gt;T (p.Phe99=)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068528\" stop=\"43068528\" display_start=\"43068528\" display_stop=\"43068528\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"A\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488638\" stop=\"44488638\" display_start=\"44488638\" display_stop=\"44488638\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"A\"/>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12403C&gt;T\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12403C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.297C&gt;T\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.297C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-750C&gt;T\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-750C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Phe99=\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Phe99=</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068528G&gt;A\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068528G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488638G&gt;A\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488638G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Phe99=\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Phe99=</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12403C&gt;T\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12403C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.297C&gt;T\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.297C&gt;T</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef Type=\"rs\" ID=\"749697783\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.297C&gt;T\" SOid=\"SO:0001819\" Function=\"synonymous variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-750C&gt;T\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n    <AlleleFrequencyList>\\n      <AlleleFrequency Value=\"0.00003\" Type=\"ExAC\" MinorAllele=\"A\"/>\\n    </AlleleFrequencyList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"519588\" VariationName=\"NM_000071.2(CBS):c.297C&gt;T (p.Phe99=)\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.297C&gt;T (p.Phe99=) AND Cardiovascular phenotype\">RCV000618182</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2016-08-04\">\\n        <Description>Likely benign</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\">\\n          <XRefList>\\n            <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"Ambry Genetics\" OrgID=\"61756\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2018-03-19\" Accession=\"SCV000738490\" Version=\"1\" mtID=\"1437132\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\">\\n            <XRefList>\\n              <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2016-08-04\">\\n          <Description>Likely benign</Description>\\n          <Comment DataSource=\"Ambry Genetics\" Type=\"public\">Lines of evidence used in support of classification: Synonymous alterations with insufficient evidence to classify as benign</Comment>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>Ambry Autosomal Dominant and X-Linked criteria (10/2015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/aH4uL7vL/mid-7377_ambry_classification_scheme_oct_2015.pdf</URL>\\n              <CitationText>mid-7377_ambry_classification_scheme_oct_2015.pdf</CitationText>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>Ambry Major Novel Clinvar Cardio to December 2017 1_24</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary Individuals=\"1\">\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"61756\" SubmitterName=\"Ambry Genetics\" OrganizationCategory=\"laboratory\" Individuals=\"1\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"61756\" SubmitterName=\"Ambry Genetics\" OrganizationCategory=\"laboratory\" Individuals=\"1\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"370129\" VariationName=\"NM_000071.2(CBS):c.316+1G&gt;A\" DateCreated=\"2017-01-04\" VariationType=\"Simple\" DateLastUpdated=\"2018-04-09\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"358645\">\\n    <Name>NM_000071.2(CBS):c.316+1G&gt;A</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068508\" stop=\"43068508\" display_start=\"43068508\" display_stop=\"43068508\" variantLength=\"1\" referenceAllele=\"C\" alternateAllele=\"T\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488618\" stop=\"44488618\" display_start=\"44488618\" display_stop=\"44488618\" variantLength=\"1\" referenceAllele=\"C\" alternateAllele=\"T\"/>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12423G&gt;A\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12423G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.316+1G&gt;A\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.316+1G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-730G&gt;A\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-730G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068508C&gt;T\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068508C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488618C&gt;T\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488618C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12423G&gt;A\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12423G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.316+1G&gt;A\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.316+1G&gt;A</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef Type=\"rs\" ID=\"1057516256\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.316+1G&gt;A\" SOid=\"SO:0001575\" Function=\"splice donor variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-730G&gt;A\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"370129\" VariationName=\"NM_000071.2(CBS):c.316+1G&gt;A\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.316+1G&gt;A AND Homocystinuria due to CBS deficiency\">RCV000411421</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2015-11-23\">\\n        <Description>Likely pathogenic</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n          <XRefList>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"Counsyl\" OrgID=\"320494\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2016-11-23\" Accession=\"SCV000485364\" Version=\"1\" mtID=\"949114\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"unknown\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n            <XRefList>\\n              <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n              <XRef ID=\"394\" DB=\"Orphanet\"/>\\n              <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2015-11-23\">\\n          <Description>Likely pathogenic</Description>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>Counsyl Autosomal and X-linked Recessive Disease Classification criteria (2015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/rb7nm21v/ar_xl2015criteria.pdf</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>Counsyl2016</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary>\\n      <AlleleOrigin Origin=\"unknown\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"320494\" SubmitterName=\"Counsyl\" OrganizationCategory=\"laboratory\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"unknown\"/>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"320494\" SubmitterName=\"Counsyl\" OrganizationCategory=\"laboratory\" Type=\"Genetic\" AlleleOrigin=\"unknown\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"340091\" VariationName=\"NM_000071.2(CBS):c.301C&gt;G (p.Leu101Val)\" DateCreated=\"2016-12-02\" VariationType=\"Simple\" DateLastUpdated=\"2018-05-03\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"350760\">\\n    <Name>NM_000071.2(CBS):c.301C&gt;G (p.Leu101Val)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068524\" stop=\"43068524\" display_start=\"43068524\" display_stop=\"43068524\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"C\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488634\" stop=\"44488634\" display_start=\"44488634\" display_stop=\"44488634\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"C\"/>\\n    <ProteinChange>L101V</ProteinChange>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12407C&gt;G\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12407C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.301C&gt;G\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.301C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-746C&gt;G\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-746C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Leu101Val\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Leu101Val</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068524G&gt;C\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068524G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488634G&gt;C\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488634G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Leu101Val\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Leu101Val</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12407C&gt;G\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12407C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.301C&gt;G\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.301C&gt;G</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef Type=\"rs\" ID=\"369644531\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.301C&gt;G\" SOid=\"SO:0001583\" Function=\"missense variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-746C&gt;G\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n    <AlleleFrequencyList>\\n      <AlleleFrequency Value=\"0.00002\" Type=\"ExAC\" MinorAllele=\"C\"/>\\n    </AlleleFrequencyList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"340091\" VariationName=\"NM_000071.2(CBS):c.301C&gt;G (p.Leu101Val)\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.301C&gt;G (p.Leu101Val) AND Homocystinuria\">RCV000310277</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2016-06-14\">\\n        <Description>Uncertain significance</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Homocystinuria\" target_id=\"15965\">\\n          <XRefList>\\n            <XRef ID=\"C0019880\" DB=\"MedGen\"/>\\n            <XRef Type=\"primary\" ID=\"HP:0002156\" DB=\"Human Phenotype Ontology\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"Illumina Clinical Services Laboratory,Illumina\" OrgID=\"504895\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"ICSL\" DateLastSubmitted=\"2016-10-18\" Accession=\"SCV000436231\" Version=\"2\" mtID=\"749752\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria\" target_id=\"15965\">\\n            <XRefList>\\n              <XRef ID=\"C0019880\" DB=\"MedGen\"/>\\n              <XRef Type=\"primary\" ID=\"HP:0002156\" DB=\"Human Phenotype Ontology\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2016-06-14\">\\n          <Description>Uncertain significance</Description>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>ICSL Variant Classification 20161018</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/4jQgNGYk/ICSL_Variant_Classification_20161018.pdf</URL>\\n              <CitationText>ICSL_Variant_Classification_20161018.pdf</CitationText>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>ICSL_2016Q4</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"504895\" SubmitterName=\"Illumina Clinical Services Laboratory,Illumina\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"ICSL\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"504895\" SubmitterName=\"Illumina Clinical Services Laboratory,Illumina\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"ICSL\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria\" target_id=\"15965\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"C0019880\" DB=\"MedGen\"/>\\n            <XRef Type=\"primary\" ID=\"HP:0002156\" DB=\"Human Phenotype Ontology\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"264609\" VariationName=\"NM_000071.2(CBS):c.210-3C&gt;T\" DateCreated=\"2016-09-29\" VariationType=\"Simple\" DateLastUpdated=\"2018-04-13\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"259106\">\\n    <Name>NM_000071.2(CBS):c.210-3C&gt;T</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068618\" stop=\"43068618\" display_start=\"43068618\" display_stop=\"43068618\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"A\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488728\" stop=\"44488728\" display_start=\"44488728\" display_stop=\"44488728\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"A\"/>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12313C&gt;T\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12313C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.210-3C&gt;T\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.210-3C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-840C&gt;T\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-840C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068618G&gt;A\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068618G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488728G&gt;A\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488728G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12313C&gt;T\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12313C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.210-3C&gt;T\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.210-3C&gt;T</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef Type=\"rs\" ID=\"562656023\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.210-3C&gt;T\" SOid=\"SO:0001627\" Function=\"intron variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-840C&gt;T\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"264609\" VariationName=\"NM_000071.2(CBS):c.210-3C&gt;T\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.210-3C&gt;T AND Cardiovascular phenotype\">RCV000243777</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2015-12-22\">\\n        <Description>Uncertain significance</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\">\\n          <XRefList>\\n            <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"Ambry Genetics\" OrgID=\"61756\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2018-03-19\" Accession=\"SCV000320641\" Version=\"3\" mtID=\"602335\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\">\\n            <XRefList>\\n              <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2015-12-22\">\\n          <Description>Uncertain significance</Description>\\n          <Comment DataSource=\"Ambry Genetics\" Type=\"public\">Lines of evidence used in support of classification: Insufficient or conflicting evidence,In silico models in agreement (benign) ,Rare (0.1%) in general population databases (dbsnp, esp, 1000 genomes)</Comment>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>Ambry Autosomal Dominant and X-Linked criteria (10/2015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/aH4uL7vL/mid-7377_ambry_classification_scheme_oct_2015.pdf</URL>\\n              <CitationText>mid-7377_ambry_classification_scheme_oct_2015.pdf</CitationText>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>Ambry Major Novel Clinvar Cardio to December 2017 1_24</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary Individuals=\"1\">\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"61756\" SubmitterName=\"Ambry Genetics\" OrganizationCategory=\"laboratory\" Individuals=\"1\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"61756\" SubmitterName=\"Ambry Genetics\" OrganizationCategory=\"laboratory\" Individuals=\"1\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"236937\" VariationName=\"NM_000071.2(CBS):c.297C&gt;G (p.Phe99Leu)\" DateCreated=\"2016-06-29\" VariationType=\"Simple\" DateLastUpdated=\"2018-05-12\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"243635\">\\n    <Name>NM_000071.2(CBS):c.297C&gt;G (p.Phe99Leu)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068528\" stop=\"43068528\" display_start=\"43068528\" display_stop=\"43068528\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"C\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488638\" stop=\"44488638\" display_start=\"44488638\" display_stop=\"44488638\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"C\"/>\\n    <ProteinChange>F99L</ProteinChange>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12403C&gt;G\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12403C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.297C&gt;G\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.297C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-750C&gt;G\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-750C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Phe99Leu\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Phe99Leu</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068528G&gt;C\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068528G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488638G&gt;C\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488638G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Phe99Leu\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Phe99Leu</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12403C&gt;G\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12403C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.297C&gt;G\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.297C&gt;G</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef Type=\"rs\" ID=\"749697783\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.297C&gt;G\" SOid=\"SO:0001583\" Function=\"missense variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-750C&gt;G\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"236937\" VariationName=\"NM_000071.2(CBS):c.297C&gt;G (p.Phe99Leu)\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.297C&gt;G (p.Phe99Leu) AND Homocystinuria due to CBS deficiency\">RCV000229409</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2015-12-06\">\\n        <Description>Uncertain significance</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n          <XRefList>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"Invitae\" OrgID=\"500031\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\" DateLastSubmitted=\"2016-06-10\" Accession=\"SCV000283385\" Version=\"2\" mtID=\"538609\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n            <XRefList>\\n              <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n              <XRef ID=\"394\" DB=\"Orphanet\"/>\\n              <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2015-12-06\">\\n          <Description>Uncertain significance</Description>\\n          <Comment DataSource=\"Invitae\" Type=\"public\">This sequence change replaces phenylalanine with leucine at codon 99 of the CBS protein (p.Phe99Leu). The phenylalanine residue is moderately conserved and there is a small physicochemical difference between phenylalanine and leucine. This variant is not present in population databases (ExAC no frequency) and has not been reported in the literature. Algorithms developed to predict the effect of missense changes on protein structure and function (SIFT, PolyPhen-2, Align-GVGD) all suggest that this variant is likely to be tolerated, but these predictions have not been confirmed by published functional studies. In summary, this is a novel missense change that is not predicted to affect protein function or cause disease. However, the evidence is insufficient at this time to prove that conclusively. It has been classified as a Variant of Uncertain Significance.</Comment>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>Invitae Variant Classification Sherloc (09022015)</Description>\\n            <Citation Type=\"general\">\\n              <ID Source=\"PubMed\">28492532</ID>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>2016June</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"500031\" SubmitterName=\"Invitae\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"500031\" SubmitterName=\"Invitae\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"212883\" VariationName=\"NM_000071.2(CBS):c.215A&gt;T (p.Lys72Ile)\" DateCreated=\"2015-10-09\" VariationType=\"Simple\" DateLastUpdated=\"2018-05-12\" SubmitterCount=\"3\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"210447\">\\n    <Name>NM_000071.2(CBS):c.215A&gt;T (p.Lys72Ile)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068610\" stop=\"43068610\" display_start=\"43068610\" display_stop=\"43068610\" variantLength=\"1\" referenceAllele=\"T\" alternateAllele=\"A\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488720\" stop=\"44488720\" display_start=\"44488720\" display_stop=\"44488720\" variantLength=\"1\" referenceAllele=\"T\" alternateAllele=\"A\"/>\\n    <OtherNameList>\\n      <Name>p.K72I:AAA&gt;ATA</Name>\\n    </OtherNameList>\\n    <ProteinChange>K72I</ProteinChange>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12321A&gt;T\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12321A&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.215A&gt;T\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.215A&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-832A&gt;T\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-832A&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Lys72Ile\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Lys72Ile</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068610T&gt;A\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068610T&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488720T&gt;A\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488720T&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Lys72Ile\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Lys72Ile</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12321A&gt;T\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12321A&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.215A&gt;T\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.215A&gt;T</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef Type=\"rs\" ID=\"192232907\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.215A&gt;T\" SOid=\"SO:0001583\" Function=\"missense variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-832A&gt;T\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n    <AlleleFrequencyList>\\n      <AlleleFrequency Value=\"0.00220\" Type=\"1000GenomesMinorAlleleFrequency\" MinorAllele=\"A\"/>\\n      <AlleleFrequency Value=\"0.00062\" Type=\"ExAC\" MinorAllele=\"A\"/>\\n    </AlleleFrequencyList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"212883\" VariationName=\"NM_000071.2(CBS):c.215A&gt;T (p.Lys72Ile)\" SubmissionCount=\"3\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.215A&gt;T (p.Lys72Ile) AND Homocystinuria due to CBS deficiency\">RCV000547149</RCV>\\n      <RCV Title=\"NM_000071.2(CBS):c.215A&gt;T (p.Lys72Ile) AND not specified\">RCV000196407</RCV>\\n      <RCV Title=\"NM_000071.2(CBS):c.215A&gt;T (p.Lys72Ile) AND Cardiovascular phenotype\">RCV000617440</RCV>\\n      <ReviewStatus>criteria provided, conflicting interpretations</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2017-05-26\">\\n        <Description>Conflicting interpretations of pathogenicity</Description>\\n        <Explanation DataSource=\"ClinVar\" Type=\"public\">Benign(1);Likely benign(1);Uncertain significance(1)</Explanation>\\n        <Comment DataSource=\"ClinVar\" Type=\"public\">Benign(1);Likely benign(1);Uncertain significance(1)</Comment>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n          <XRefList>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n        <Phenotype Name=\"not specified\" target_id=\"16789\">\\n          <XRefList>\\n            <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n        <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\">\\n          <XRefList>\\n            <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"GeneDx\" OrgID=\"26957\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2017-11-28\" Accession=\"SCV000249734\" Version=\"11\" mtID=\"464820\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"not specified\" target_id=\"16789\">\\n            <XRefList>\\n              <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2016-10-14\">\\n          <Description>Uncertain significance</Description>\\n          <Comment DataSource=\"GeneDx\" Type=\"public\">The K72I variant has not been published as apathogenic variant, nor has it been reported as a benign variant to our knowledge. However, this variant haspreviously been observed in other unrelated individuals referred for Marfan/TAAD genetic testing at GeneDx, some ofwhom also harbored likely pathogenic variants in another Marfan/TAAD-associated gene. The K72I variant is anon-conservative amino acid substitution, which is likely to impact secondary protein structure as these residues differin polarity, charge, size and/or other properties. In silico analysis predicts this variant is probably damaging to theprotein structure/function. However, this substitution occurs at a position that is not conserved across species.Furthermore, the 1000 Genomes Project and the Exome Aggregation Consortium (ExAC) report K72I was observedin 10/1,008 (0.99%) and 66/86,620 (0.77%) alleles from individuals of East Asian background, respectively.Therefore, based on the currently available information, it is unclear whether this variant is pathogenic or benign.</Comment>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>GeneDx Variant Classification (06012015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/7oynscmk/mdi-5616_26957_genedx_interprules_final_061215.pdf</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n      </Germline>\\n      <Germline SubmitterName=\"Invitae\" OrgID=\"500031\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\" DateLastSubmitted=\"2017-10-05\" Accession=\"SCV000649830\" Version=\"1\" mtID=\"1242369\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n            <XRefList>\\n              <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n              <XRef ID=\"394\" DB=\"Orphanet\"/>\\n              <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2017-02-17\">\\n          <Description>Likely benign</Description>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>Invitae Variant Classification Sherloc (09022015)</Description>\\n            <Citation Type=\"general\">\\n              <ID Source=\"PubMed\">28492532</ID>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>SUB3109251</SubmitterBatch>\\n      </Germline>\\n      <Germline SubmitterName=\"Ambry Genetics\" OrgID=\"61756\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2018-03-19\" Accession=\"SCV000738498\" Version=\"1\" mtID=\"1437140\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\">\\n            <XRefList>\\n              <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2017-05-26\">\\n          <Description>Benign</Description>\\n          <Comment DataSource=\"Ambry Genetics\" Type=\"public\">Lines of evidence used in support of classification: General population or sub-population frequency is too high to be a pathogenic mutation based on disease/syndrome prevalence and penetrance</Comment>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>Ambry Autosomal Dominant and X-Linked criteria (3/2017)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/10Jq49tz/AD_XL_ClassificationScheme_03.2017_Ambry.pdf</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>Ambry Major Novel Clinvar Cardio to December 2017 1_24</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary Individuals=\"1\">\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"61756\" SubmitterName=\"Ambry Genetics\" OrganizationCategory=\"laboratory\" Individuals=\"1\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n    <SubmitterSummary OrgID=\"26957\" SubmitterName=\"GeneDx\" OrganizationCategory=\"laboratory\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n    <SubmitterSummary OrgID=\"500031\" SubmitterName=\"Invitae\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"61756\" SubmitterName=\"Ambry Genetics\" OrganizationCategory=\"laboratory\" Individuals=\"1\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n    <Observation OrgID=\"26957\" SubmitterName=\"GeneDx\" OrganizationCategory=\"laboratory\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"not specified\" target_id=\"16789\" AffectedStatus=\"yes\">\\n          <XRefList>\\n            <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n    <Observation OrgID=\"500031\" SubmitterName=\"Invitae\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"212874\" VariationName=\"NM_000071.2(CBS):c.253G&gt;A (p.Gly85Arg)\" DateCreated=\"2015-10-09\" VariationType=\"Simple\" DateLastUpdated=\"2018-05-18\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"210446\">\\n    <Name>NM_000071.2(CBS):c.253G&gt;A (p.Gly85Arg)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068572\" stop=\"43068572\" display_start=\"43068572\" display_stop=\"43068572\" variantLength=\"1\" referenceAllele=\"C\" alternateAllele=\"T\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488682\" stop=\"44488682\" display_start=\"44488682\" display_stop=\"44488682\" variantLength=\"1\" referenceAllele=\"C\" alternateAllele=\"T\"/>\\n    <OtherNameList>\\n      <Name>p.G85R:GGG&gt;AGG</Name>\\n    </OtherNameList>\\n    <ProteinChange>G85R</ProteinChange>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12359G&gt;A\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12359G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.253G&gt;A\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.253G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-794G&gt;A\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-794G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Gly85Arg\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Gly85Arg</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068572C&gt;T\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068572C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488682C&gt;T\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488682C&gt;T</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Gly85Arg\">P35520:p.Gly85Arg</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Gly85Arg\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Gly85Arg</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12359G&gt;A\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12359G&gt;A</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.253G&gt;A\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.253G&gt;A</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef ID=\"P35520#VAR_008051\" DB=\"UniProtKB\"/>\\n      <XRef Type=\"rs\" ID=\"863223435\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.253G&gt;A\" SOid=\"SO:0001583\" Function=\"missense variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-794G&gt;A\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"212874\" VariationName=\"NM_000071.2(CBS):c.253G&gt;A (p.Gly85Arg)\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.253G&gt;A (p.Gly85Arg) AND not provided\">RCV000195506</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2016-12-07\">\\n        <Description>Pathogenic</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"not provided\" target_id=\"17556\">\\n          <XRefList>\\n            <XRef ID=\"CN517202\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"GeneDx\" OrgID=\"26957\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2017-11-28\" Accession=\"SCV000249722\" Version=\"7\" mtID=\"464808\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"not provided\" target_id=\"17556\">\\n            <XRefList>\\n              <XRef ID=\"CN517202\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2016-12-07\">\\n          <Description>Pathogenic</Description>\\n          <Comment DataSource=\"GeneDx\" Type=\"public\">The G85R pathogenic variant in the CBS gene has been reported previously either in the homozygous state or with another CBS pathogenic variant in multiple individuals with homocystinuria (Maclean et al., 2002; De Lucca et al, 2004; Cozar et al, 2011; Casique et al., 2013; Karaca et al., 2014). Functional studies in E. coli demonstrated that the G85R variant was associated with no detectable CBS enzyme activity (Maclean et al., 2002). The G85R variant was not observed in approximately 6,500 individuals of European and African American ancestry in the NHLBI Exome Sequencing Project, indicating it is not a common benign variant in these populations. The G85R variant is a non-conservative amino acid substitution and occurs at a position that is conserved across species. We interpret G85R as a pathogenic variant.</Comment>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>GeneDx Variant Classification (06012015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/7oynscmk/mdi-5616_26957_genedx_interprules_final_061215.pdf</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"26957\" SubmitterName=\"GeneDx\" OrganizationCategory=\"laboratory\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"26957\" SubmitterName=\"GeneDx\" OrganizationCategory=\"laboratory\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"not provided\" target_id=\"17556\" AffectedStatus=\"yes\">\\n          <XRefList>\\n            <XRef ID=\"CN517202\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"189185\" VariationName=\"NM_000071.2(CBS):c.302T&gt;C (p.Leu101Pro)\" DateCreated=\"2015-03-28\" VariationType=\"Simple\" DateLastUpdated=\"2018-04-09\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"187062\">\\n    <Name>NM_000071.2(CBS):c.302T&gt;C (p.Leu101Pro)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068523\" stop=\"43068523\" display_start=\"43068523\" display_stop=\"43068523\" variantLength=\"1\" referenceAllele=\"A\" alternateAllele=\"G\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488633\" stop=\"44488633\" display_start=\"44488633\" display_stop=\"44488633\" variantLength=\"1\" referenceAllele=\"A\" alternateAllele=\"G\"/>\\n    <ProteinChange>L101P</ProteinChange>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12408T&gt;C\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12408T&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.302T&gt;C\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.302T&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-745T&gt;C\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-745T&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Leu101Pro\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Leu101Pro</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068523A&gt;G\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068523A&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488633A&gt;G\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488633A&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Leu101Pro\">P35520:p.Leu101Pro</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Leu101Pro\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Leu101Pro</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12408T&gt;C\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12408T&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.302T&gt;C\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.302T&gt;C</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef ID=\"P35520#VAR_021791\" DB=\"UniProtKB\"/>\\n      <XRef Type=\"rs\" ID=\"786204757\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.302T&gt;C\" SOid=\"SO:0001583\" Function=\"missense variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-745T&gt;C\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"189185\" VariationName=\"NM_000071.2(CBS):c.302T&gt;C (p.Leu101Pro)\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.302T&gt;C (p.Leu101Pro) AND Homocystinuria due to CBS deficiency\">RCV000169617</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2015-02-18\">\\n        <Description>Likely pathogenic</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n          <XRefList>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"Counsyl\" OrgID=\"320494\" OrganizationCategory=\"laboratory\" ModeOfInheritance=\"Autosomal recessive inheritance\" DateLastSubmitted=\"2015-03-11\" Accession=\"SCV000221143\" Version=\"1\" mtID=\"397899\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"unknown\" MethodType=\"literature only\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n            <XRefList>\\n              <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n              <XRef ID=\"394\" DB=\"Orphanet\"/>\\n              <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2015-02-18\">\\n          <Description>Likely pathogenic</Description>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">12686134</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">9889017</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">17069888</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">14635102</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">22267502</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">20455263</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">12124992</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">20066033</ID>\\n          </Citation>\\n          <Method>literature only</Method>\\n          <AssertionMethod>\\n            <Description>Counsyl Autosomal and X-linked Recessive Disease Classification criteria (2015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/rb7nm21v/ar_xl2015criteria.pdf</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>Counsyl_2015_03_01</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary>\\n      <AlleleOrigin Origin=\"unknown\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"320494\" SubmitterName=\"Counsyl\" OrganizationCategory=\"laboratory\">\\n      <Method>\\n        <MethodName>literature only</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"unknown\"/>\\n      <CitationList>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">12124992</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">12686134</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">14635102</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17069888</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20066033</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20455263</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22267502</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">9889017</ID>\\n        </Citation>\\n      </CitationList>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"320494\" SubmitterName=\"Counsyl\" OrganizationCategory=\"laboratory\" Type=\"Genetic\" AlleleOrigin=\"unknown\">\\n      <Method>\\n        <MethodName>literature only</MethodName>\\n      </Method>\\n      <CitationList>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">12686134</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">9889017</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17069888</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">14635102</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22267502</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20455263</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">12124992</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20066033</ID>\\n        </Citation>\\n      </CitationList>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"188989\" VariationName=\"NM_000071.2(CBS):c.306G&gt;C (p.Lys102Asn)\" DateCreated=\"2015-03-28\" VariationType=\"Simple\" DateLastUpdated=\"2018-04-09\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"187061\">\\n    <Name>NM_000071.2(CBS):c.306G&gt;C (p.Lys102Asn)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068519\" stop=\"43068519\" display_start=\"43068519\" display_stop=\"43068519\" variantLength=\"1\" referenceAllele=\"C\" alternateAllele=\"G\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488629\" stop=\"44488629\" display_start=\"44488629\" display_stop=\"44488629\" variantLength=\"1\" referenceAllele=\"C\" alternateAllele=\"G\"/>\\n    <ProteinChange>K102N</ProteinChange>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12412G&gt;C\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12412G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.306G&gt;C\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.306G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-741G&gt;C\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-741G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Lys102Asn\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Lys102Asn</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068519C&gt;G\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068519C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488629C&gt;G\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488629C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Lys102Asn\">P35520:p.Lys102Asn</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Lys102Asn\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Lys102Asn</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12412G&gt;C\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12412G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.306G&gt;C\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.306G&gt;C</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef ID=\"P35520#VAR_002173\" DB=\"UniProtKB\"/>\\n      <XRef Type=\"rs\" ID=\"786204609\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.306G&gt;C\" SOid=\"SO:0001583\" Function=\"missense variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-741G&gt;C\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"188989\" VariationName=\"NM_000071.2(CBS):c.306G&gt;C (p.Lys102Asn)\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.306G&gt;C (p.Lys102Asn) AND Homocystinuria due to CBS deficiency\">RCV000169368</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2014-09-26\">\\n        <Description>Likely pathogenic</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n          <XRefList>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"Counsyl\" OrgID=\"320494\" OrganizationCategory=\"laboratory\" ModeOfInheritance=\"Autosomal recessive inheritance\" DateLastSubmitted=\"2015-03-11\" Accession=\"SCV000220743\" Version=\"1\" mtID=\"397499\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"unknown\" MethodType=\"literature only\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n            <XRefList>\\n              <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n              <XRef ID=\"394\" DB=\"Orphanet\"/>\\n              <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2014-09-26\">\\n          <Description>Likely pathogenic</Description>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">12686134</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">17352495</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">7981678</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">17069888</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">22267502</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">20506325</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">20490928</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">7967489</ID>\\n          </Citation>\\n          <Method>literature only</Method>\\n          <AssertionMethod>\\n            <Description>Counsyl Autosomal and X-linked Recessive Disease Classification criteria (2015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/rb7nm21v/ar_xl2015criteria.pdf</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>Counsyl_2015_03_01</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary>\\n      <AlleleOrigin Origin=\"unknown\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"320494\" SubmitterName=\"Counsyl\" OrganizationCategory=\"laboratory\">\\n      <Method>\\n        <MethodName>literature only</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"unknown\"/>\\n      <CitationList>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">12686134</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17069888</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17352495</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20490928</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20506325</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22267502</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">7967489</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">7981678</ID>\\n        </Citation>\\n      </CitationList>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"320494\" SubmitterName=\"Counsyl\" OrganizationCategory=\"laboratory\" Type=\"Genetic\" AlleleOrigin=\"unknown\">\\n      <Method>\\n        <MethodName>literature only</MethodName>\\n      </Method>\\n      <CitationList>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">12686134</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17352495</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">7981678</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17069888</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22267502</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20506325</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20490928</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">7967489</ID>\\n        </Citation>\\n      </CitationList>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"188988\" VariationName=\"NM_000071.2(CBS):c.233C&gt;G (p.Pro78Arg)\" DateCreated=\"2015-03-28\" VariationType=\"Simple\" DateLastUpdated=\"2018-04-09\" SubmitterCount=\"1\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"187063\">\\n    <Name>NM_000071.2(CBS):c.233C&gt;G (p.Pro78Arg)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068592\" stop=\"43068592\" display_start=\"43068592\" display_stop=\"43068592\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"C\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488702\" stop=\"44488702\" display_start=\"44488702\" display_stop=\"44488702\" variantLength=\"1\" referenceAllele=\"G\" alternateAllele=\"C\"/>\\n    <ProteinChange>P78R</ProteinChange>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12339C&gt;G\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12339C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.233C&gt;G\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.233C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-814C&gt;G\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-814C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Pro78Arg\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Pro78Arg</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068592G&gt;C\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068592G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488702G&gt;C\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488702G&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Pro78Arg\">P35520:p.Pro78Arg</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Pro78Arg\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Pro78Arg</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12339C&gt;G\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12339C&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.233C&gt;G\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.233C&gt;G</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef ID=\"P35520#VAR_002171\" DB=\"UniProtKB\"/>\\n      <XRef Type=\"rs\" ID=\"786204608\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.233C&gt;G\" SOid=\"SO:0001583\" Function=\"missense variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-814C&gt;G\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"188988\" VariationName=\"NM_000071.2(CBS):c.233C&gt;G (p.Pro78Arg)\" SubmissionCount=\"1\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.233C&gt;G (p.Pro78Arg) AND Homocystinuria due to CBS deficiency\">RCV000169367</RCV>\\n      <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2014-09-26\">\\n        <Description>Likely pathogenic</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n          <XRefList>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"Counsyl\" OrgID=\"320494\" OrganizationCategory=\"laboratory\" ModeOfInheritance=\"Autosomal recessive inheritance\" DateLastSubmitted=\"2015-03-11\" Accession=\"SCV000220742\" Version=\"1\" mtID=\"397498\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"unknown\" MethodType=\"literature only\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n            <XRefList>\\n              <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n              <XRef ID=\"394\" DB=\"Orphanet\"/>\\n              <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2014-09-26\">\\n          <Description>Likely pathogenic</Description>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">12686134</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">22985361</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">17352495</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">7981678</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">22612060</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">17069888</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">22267502</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">20506325</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">20490928</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">20308073</ID>\\n          </Citation>\\n          <Citation Type=\"general\">\\n            <ID Source=\"PubMed\">7967489</ID>\\n          </Citation>\\n          <Method>literature only</Method>\\n          <AssertionMethod>\\n            <Description>Counsyl Autosomal and X-linked Recessive Disease Classification criteria (2015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/rb7nm21v/ar_xl2015criteria.pdf</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>Counsyl_2015_03_01</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary>\\n      <AlleleOrigin Origin=\"unknown\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"320494\" SubmitterName=\"Counsyl\" OrganizationCategory=\"laboratory\">\\n      <Method>\\n        <MethodName>literature only</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"unknown\"/>\\n      <CitationList>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">12686134</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17069888</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17352495</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20308073</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20490928</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20506325</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22267502</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22612060</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22985361</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">7967489</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">7981678</ID>\\n        </Citation>\\n      </CitationList>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"320494\" SubmitterName=\"Counsyl\" OrganizationCategory=\"laboratory\" Type=\"Genetic\" AlleleOrigin=\"unknown\">\\n      <Method>\\n        <MethodName>literature only</MethodName>\\n      </Method>\\n      <CitationList>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">12686134</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22985361</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17352495</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">7981678</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22612060</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">17069888</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">22267502</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20506325</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20490928</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">20308073</ID>\\n        </Citation>\\n        <Citation Type=\"general\">\\n          <ID Source=\"PubMed\">7967489</ID>\\n        </Citation>\\n      </CitationList>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport><VariationReport VariationID=\"136677\" VariationName=\"NM_000071.2(CBS):c.304A&gt;C (p.Lys102Gln)\" DateCreated=\"2014-06-18\" VariationType=\"Simple\" DateLastUpdated=\"2018-05-26\" SubmitterCount=\"7\">\\n  <Species TaxonomyId=\"9606\">human</Species>\\n  <GeneList GeneCount=\"1\">\\n    <Gene GeneID=\"875\" Symbol=\"CBS\" FullName=\"cystathionine-beta-synthase\" HGNCID=\"HGNC:1550\" strand=\"-\" Type=\"submitted\" RelationshipType=\"within single gene\">\\n      <OMIM>613381</OMIM>\\n    </Gene>\\n  </GeneList>\\n  <Allele AlleleID=\"140380\">\\n    <Name>NM_000071.2(CBS):c.304A&gt;C (p.Lys102Gln)</Name>\\n    <VariantType>single nucleotide variant</VariantType>\\n    <CytogeneticLocation>21q22.3</CytogeneticLocation>\\n    <SequenceLocation Assembly=\"GRCh38\" AssemblyAccessionVersion=\"GCF_000001405.33\" AssemblyStatus=\"current\" Chr=\"21\" Accession=\"NC_000021.9\" start=\"43068521\" stop=\"43068521\" display_start=\"43068521\" display_stop=\"43068521\" variantLength=\"1\" referenceAllele=\"T\" alternateAllele=\"G\"/>\\n    <SequenceLocation Assembly=\"GRCh37\" AssemblyAccessionVersion=\"GCF_000001405.25\" AssemblyStatus=\"previous\" Chr=\"21\" Accession=\"NC_000021.8\" start=\"44488631\" stop=\"44488631\" display_start=\"44488631\" display_stop=\"44488631\" variantLength=\"1\" referenceAllele=\"T\" alternateAllele=\"G\"/>\\n    <OtherNameList>\\n      <Name>p.K102Q:AAG&gt;CAG</Name>\\n    </OtherNameList>\\n    <ProteinChange>K102Q</ProteinChange>\\n    <HGVSlist>\\n      <HGVS Type=\"HGVS, genomic, RefSeqGene\" Change=\"g.12410A&gt;C\" AccessionVersion=\"NG_008938.1\" Accession=\"NG_008938\" Version=\"1\">NG_008938.1:g.12410A&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.304A&gt;C\" AccessionVersion=\"NM_000071.2\" Accession=\"NM_000071\" Version=\"2\">NM_000071.2:c.304A&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.304A&gt;C\" AccessionVersion=\"NM_001178009.2\" Accession=\"NM_001178009\" Version=\"2\">NM_001178009.2:c.304A&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, RefSeq\" Change=\"c.-743A&gt;C\" AccessionVersion=\"NM_001321072.1\" Accession=\"NM_001321072\" Version=\"1\">NM_001321072.1:c.-743A&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Lys102Gln\" AccessionVersion=\"NP_000062.1\" Accession=\"NP_000062\" Version=\"1\">NP_000062.1:p.Lys102Gln</HGVS>\\n      <HGVS Type=\"HGVS, protein, RefSeq\" Change=\"p.Lys102Gln\" AccessionVersion=\"NP_001171480.1\" Accession=\"NP_001171480\" Version=\"1\">NP_001171480.1:p.Lys102Gln</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level\" Change=\"g.43068521T&gt;G\" AccessionVersion=\"NC_000021.9\" Accession=\"NC_000021\" Version=\"9\" Assembly=\"GRCh38\">NC_000021.9:g.43068521T&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, genomic, top level, previous\" Change=\"g.44488631T&gt;G\" AccessionVersion=\"NC_000021.8\" Accession=\"NC_000021\" Version=\"8\" Assembly=\"GRCh37\">NC_000021.8:g.44488631T&gt;G</HGVS>\\n      <HGVS Type=\"HGVS, previous\" Change=\"c.304A&gt;C\" AccessionVersion=\"NM_001178009.1\" Accession=\"NM_001178009\" Version=\"1\">NM_001178009.1:c.304A&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Lys102Gln\">P35520:p.Lys102Gln</HGVS>\\n      <HGVS Type=\"HGVS, protein\" Change=\"p.Lys102Gln\" AccessionVersion=\"LRG_777p1\" Accession=\"LRG_777p1\">LRG_777p1:p.Lys102Gln</HGVS>\\n      <HGVS Type=\"HGVS, genomic, LRG\" Change=\"g.12410A&gt;C\" AccessionVersion=\"LRG_777\" Accession=\"LRG_777\">LRG_777:g.12410A&gt;C</HGVS>\\n      <HGVS Type=\"HGVS, coding, LRG\" Change=\"c.304A&gt;C\" AccessionVersion=\"LRG_777t1\" Accession=\"LRG_777t1\">LRG_777t1:c.304A&gt;C</HGVS>\\n    </HGVSlist>\\n    <XRefList>\\n      <XRef ID=\"P35520#VAR_008052\" DB=\"UniProtKB\"/>\\n      <XRef Type=\"rs\" ID=\"34040148\" DB=\"dbSNP\"/>\\n    </XRefList>\\n    <MolecularConsequenceList>\\n      <MolecularConsequence HGVS=\"NM_000071.2:c.304A&gt;C\" SOid=\"SO:0001583\" Function=\"missense variant\"/>\\n      <MolecularConsequence HGVS=\"NM_001321072.1:c.-743A&gt;C\" SOid=\"SO:0001636\" Function=\"2KB upstream variant\"/>\\n    </MolecularConsequenceList>\\n    <AlleleFrequencyList>\\n      <AlleleFrequency Value=\"0.01180\" Type=\"1000GenomesMinorAlleleFrequency\" MinorAllele=\"G\"/>\\n      <AlleleFrequency Value=\"0.00363\" Type=\"ExAC\" MinorAllele=\"G\"/>\\n    </AlleleFrequencyList>\\n  </Allele>\\n  <ObservationList>\\n    <Observation VariationID=\"136677\" VariationName=\"NM_000071.2(CBS):c.304A&gt;C (p.Lys102Gln)\" SubmissionCount=\"7\" ObservationType=\"primary\">\\n      <RCV Title=\"NM_000071.2(CBS):c.304A&gt;C (p.Lys102Gln) AND Homocystinuria due to CBS deficiency\">RCV000233317</RCV>\\n      <RCV Title=\"NM_000071.2(CBS):c.304A&gt;C (p.Lys102Gln) AND not provided\">RCV000224394</RCV>\\n      <RCV Title=\"NM_000071.2(CBS):c.304A&gt;C (p.Lys102Gln) AND not specified\">RCV000178036</RCV>\\n      <RCV Title=\"NM_000071.2(CBS):c.304A&gt;C (p.Lys102Gln) AND Homocystinuria\">RCV000394063</RCV>\\n      <RCV Title=\"NM_000071.2(CBS):c.304A&gt;C (p.Lys102Gln) AND Cardiovascular phenotype\">RCV000621680</RCV>\\n      <ReviewStatus>criteria provided, multiple submitters, no conflicts</ReviewStatus>\\n      <ClinicalSignificance DateLastEvaluated=\"2017-11-21\">\\n        <Description>Benign/Likely benign</Description>\\n      </ClinicalSignificance>\\n      <PhenotypeList>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n          <XRefList>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n        <Phenotype Name=\"Homocystinuria\" target_id=\"15965\">\\n          <XRefList>\\n            <XRef ID=\"C0019880\" DB=\"MedGen\"/>\\n            <XRef Type=\"primary\" ID=\"HP:0002156\" DB=\"Human Phenotype Ontology\"/>\\n          </XRefList>\\n        </Phenotype>\\n        <Phenotype Name=\"not specified\" target_id=\"16789\">\\n          <XRefList>\\n            <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n        <Phenotype Name=\"not provided\" target_id=\"17556\">\\n          <XRefList>\\n            <XRef ID=\"CN517202\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n        <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\">\\n          <XRefList>\\n            <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </PhenotypeList>\\n    </Observation>\\n  </ObservationList>\\n  <ClinicalAssertionList>\\n    <GermlineList>\\n      <Germline SubmitterName=\"GeneDx\" OrgID=\"26957\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2018-03-26\" Accession=\"SCV000167573\" Version=\"12\" mtID=\"289497\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"not specified\" target_id=\"16789\">\\n            <XRefList>\\n              <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2013-01-20\">\\n          <Description>Benign</Description>\\n          <Comment DataSource=\"GeneDx\" Type=\"public\">This variant is considered likely benign or benign based on one or more of the following criteria: it is a conservative change, it occurs at a poorly conserved position in the protein, it is predicted to be benign by multiple in silico algorithms, and/or has population frequency not consistent with disease.</Comment>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>GeneDX Variant Classification (06012015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/7oynscmk/mdi-5616_26957_genedx_interprules_final_061215.pdf</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>SUB3839901</SubmitterBatch>\\n      </Germline>\\n      <Germline SubmitterName=\"EGL Genetic Diagnostics,Eurofins Clinical Diagnostics\" OrgID=\"500060\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2017-06-30\" Accession=\"SCV000230022\" Version=\"4\" mtID=\"418342\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"not specified\" target_id=\"16789\">\\n            <XRefList>\\n              <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2015-02-03\">\\n          <Description>Benign</Description>\\n          <Citation Type=\"general\">\\n            <URL>http://www.egl-eurofins.com/emvclass/emvclass.php?approved_symbol=CBS</URL>\\n          </Citation>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>EGL Classification Definitions</Description>\\n            <Citation Type=\"general\">\\n              <URL>http://www.egl-eurofins.com/emvclass/EGLClassificationDefinitions.php</URL>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>EGL005</SubmitterBatch>\\n      </Germline>\\n      <Germline SubmitterName=\"Laboratory for Molecular Medicine,Partners HealthCare Personalized Medicine\" OrgID=\"21766\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2018-02-23\" Accession=\"SCV000268833\" Version=\"2\" mtID=\"505107\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"not specified\" target_id=\"16789\">\\n            <XRefList>\\n              <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2015-07-16\">\\n          <Description>Benign</Description>\\n          <Comment DataSource=\"Laboratory for Molecular Medicine,Partners HealthCare Personalized Medicine\" Type=\"public\">p.Lys102Gln in exon 4 of CBS: This variant is not expected to have clinical significance because it has been identified in 4% (415/10368) of African chromosomes by the Exome Aggregation Consortium (ExAC,http://exac.broadinstitute.org; dbSNP rs34040148).</Comment>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>LMM Criteria</Description>\\n            <Citation Type=\"general\">\\n              <ID Source=\"PubMed\">24033266</ID>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>SUB3719835</SubmitterBatch>\\n      </Germline>\\n      <Germline SubmitterName=\"Center for Pediatric Genomic Medicine,Children\\'s Mercy Hospital and Clinics\" OrgID=\"505698\" OrganizationCategory=\"clinic\" DateLastSubmitted=\"2016-05-19\" Accession=\"SCV000281546\" Version=\"1\" mtID=\"530654\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"not provided\" target_id=\"17556\">\\n            <XRefList>\\n              <XRef ID=\"CN517202\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2015-05-21\">\\n          <Description>Benign</Description>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>ACMG Guidelines, 2015</Description>\\n            <Citation Type=\"practice guideline\" Abbrev=\"ACMG, 2015\">\\n              <ID Source=\"PubMed\">25741868</ID>\\n              <ID Source=\"DOI\">10.1038/gim.2015.30</ID>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n      </Germline>\\n      <Germline SubmitterName=\"Invitae\" OrgID=\"500031\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\" DateLastSubmitted=\"2018-04-02\" Accession=\"SCV000283386\" Version=\"4\" mtID=\"538610\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\">\\n            <XRefList>\\n              <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n              <XRef ID=\"394\" DB=\"Orphanet\"/>\\n              <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2017-11-21\">\\n          <Description>Benign</Description>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>Nykamp K et al. (Genet Med 2017)</Description>\\n            <Citation Type=\"general\">\\n              <ID Source=\"PubMed\">28492532</ID>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>SUB3864685</SubmitterBatch>\\n      </Germline>\\n      <Germline SubmitterName=\"Illumina Clinical Services Laboratory,Illumina\" OrgID=\"504895\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"ICSL\" DateLastSubmitted=\"2016-10-18\" Accession=\"SCV000436230\" Version=\"2\" mtID=\"749751\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Homocystinuria\" target_id=\"15965\">\\n            <XRefList>\\n              <XRef ID=\"C0019880\" DB=\"MedGen\"/>\\n              <XRef Type=\"primary\" ID=\"HP:0002156\" DB=\"Human Phenotype Ontology\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2016-06-14\">\\n          <Description>Likely benign</Description>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>ICSL Variant Classification 20161018</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/4jQgNGYk/ICSL_Variant_Classification_20161018.pdf</URL>\\n              <CitationText>ICSL_Variant_Classification_20161018.pdf</CitationText>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>ICSL_2016Q4</SubmitterBatch>\\n      </Germline>\\n      <Germline SubmitterName=\"Ambry Genetics\" OrgID=\"61756\" OrganizationCategory=\"laboratory\" DateLastSubmitted=\"2018-03-19\" Accession=\"SCV000738386\" Version=\"1\" mtID=\"1437008\">\\n        <AlleleOriginList>\\n          <AlleleOrigin Origin=\"germline\" MethodType=\"clinical testing\"/>\\n        </AlleleOriginList>\\n        <PhenotypeList>\\n          <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\">\\n            <XRefList>\\n              <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n            </XRefList>\\n          </Phenotype>\\n        </PhenotypeList>\\n        <ReviewStatus>criteria provided, single submitter</ReviewStatus>\\n        <ClinicalSignificance DateLastEvaluated=\"2015-08-25\">\\n          <Description>Benign</Description>\\n          <Method>clinical testing</Method>\\n          <AssertionMethod>\\n            <Description>Ambry Autosomal Dominant and X-Linked criteria (10/2015)</Description>\\n            <Citation Type=\"general\">\\n              <URL>https://submit.ncbi.nlm.nih.gov/ft/byid/aH4uL7vL/mid-7377_ambry_classification_scheme_oct_2015.pdf</URL>\\n              <CitationText>mid-7377_ambry_classification_scheme_oct_2015.pdf</CitationText>\\n            </Citation>\\n          </AssertionMethod>\\n        </ClinicalSignificance>\\n        <SubmitterBatch>Ambry Major Novel Clinvar Cardio to December 2017 1_24</SubmitterBatch>\\n      </Germline>\\n    </GermlineList>\\n  </ClinicalAssertionList>\\n  <SummaryEvidence>\\n    <Summary Families=\"1\" Individuals=\"3\">\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </Summary>\\n    <SubmitterSummary OrgID=\"61756\" SubmitterName=\"Ambry Genetics\" OrganizationCategory=\"laboratory\" Individuals=\"1\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n    <SubmitterSummary OrgID=\"505698\" SubmitterName=\"Center for Pediatric Genomic Medicine,Children\\'s Mercy Hospital and Clinics\" OrganizationCategory=\"clinic\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n    <SubmitterSummary OrgID=\"500060\" SubmitterName=\"EGL Genetic Diagnostics,Eurofins Clinical Diagnostics\" OrganizationCategory=\"laboratory\" Individuals=\"1\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n      <CitationList>\\n        <Citation Type=\"general\">\\n          <URL>http://www.egl-eurofins.com/emvclass/emvclass.php?approved_symbol=CBS</URL>\\n        </Citation>\\n      </CitationList>\\n    </SubmitterSummary>\\n    <SubmitterSummary OrgID=\"26957\" SubmitterName=\"GeneDx\" OrganizationCategory=\"laboratory\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n    <SubmitterSummary OrgID=\"504895\" SubmitterName=\"Illumina Clinical Services Laboratory,Illumina\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"ICSL\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n    <SubmitterSummary OrgID=\"500031\" SubmitterName=\"Invitae\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n    <SubmitterSummary OrgID=\"21766\" SubmitterName=\"Laboratory for Molecular Medicine,Partners HealthCare Personalized Medicine\" OrganizationCategory=\"laboratory\" Families=\"1\" Individuals=\"1\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <AlleleOrigin Origin=\"germline\"/>\\n    </SubmitterSummary>\\n  </SummaryEvidence>\\n  <SupportingObservations>\\n    <Observation OrgID=\"61756\" SubmitterName=\"Ambry Genetics\" OrganizationCategory=\"laboratory\" Individuals=\"1\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Cardiovascular phenotype\" target_id=\"33389\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"CN230736\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n    <Observation OrgID=\"505698\" SubmitterName=\"Center for Pediatric Genomic Medicine,Children\\'s Mercy Hospital and Clinics\" OrganizationCategory=\"clinic\" Type=\"Genetic\" AlleleOrigin=\"germline\" AlleleFrequency=\"0.003587\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"not provided\" target_id=\"17556\" AffectedStatus=\"not provided\">\\n          <XRefList>\\n            <XRef ID=\"CN517202\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n    <Observation OrgID=\"500060\" SubmitterName=\"EGL Genetic Diagnostics,Eurofins Clinical Diagnostics\" OrganizationCategory=\"laboratory\" Individuals=\"1\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <CitationList>\\n        <Citation Type=\"general\">\\n          <URL>http://www.egl-eurofins.com/emvclass/emvclass.php?approved_symbol=CBS</URL>\\n        </Citation>\\n      </CitationList>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"not specified\" target_id=\"16789\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n    <Observation OrgID=\"26957\" SubmitterName=\"GeneDx\" OrganizationCategory=\"laboratory\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"not specified\" target_id=\"16789\" AffectedStatus=\"yes\">\\n          <XRefList>\\n            <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n    <Observation OrgID=\"504895\" SubmitterName=\"Illumina Clinical Services Laboratory,Illumina\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"ICSL\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria\" target_id=\"15965\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"C0019880\" DB=\"MedGen\"/>\\n            <XRef Type=\"primary\" ID=\"HP:0002156\" DB=\"Human Phenotype Ontology\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n    <Observation OrgID=\"500031\" SubmitterName=\"Invitae\" OrganizationCategory=\"laboratory\" OrgAbbreviation=\"Invitae\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"Homocystinuria due to CBS deficiency\" target_id=\"1656\" AffectedStatus=\"unknown\">\\n          <XRefList>\\n            <XRef ID=\"394\" DB=\"Orphanet\"/>\\n            <XRef ID=\"C3150344\" DB=\"MedGen\"/>\\n            <XRef Type=\"MIM\" ID=\"236200\" DB=\"OMIM\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n    <Observation OrgID=\"21766\" SubmitterName=\"Laboratory for Molecular Medicine,Partners HealthCare Personalized Medicine\" OrganizationCategory=\"laboratory\" Families=\"1\" Individuals=\"1\" Type=\"Genetic\" AlleleOrigin=\"germline\">\\n      <Method>\\n        <MethodName>clinical testing</MethodName>\\n      </Method>\\n      <ObservedPhenotypes>\\n        <Phenotype Name=\"not specified\" target_id=\"16789\" AffectedStatus=\"not provided\">\\n          <XRefList>\\n            <XRef ID=\"CN169374\" DB=\"MedGen\"/>\\n          </XRefList>\\n        </Phenotype>\\n      </ObservedPhenotypes>\\n    </Observation>\\n  </SupportingObservations>\\n</VariationReport>\\n</ClinVarResult-Set>'"
      ]
     },
     "execution_count": 102,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "handle.read()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {},
   "outputs": [
    {
     "ename": "TypeError",
     "evalue": "'str' object is not an iterator",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-96-99bd7a0c47e0>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m()\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mnext\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mres\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;31mTypeError\u001b[0m: 'str' object is not an iterator"
     ]
    }
   ],
   "source": [
    "res"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
